docs/topics/modules/viewing_build_pipeline_logs.adoc

```diff
@@ -0,0 +1,14 @@
+[id="viewing_build_pipeline_logs"]
+= (Optional) Viewing the build pipeline logs
+
+Optionally, while you wait for the pipeline build, you can view the build details in the Jenkins log. For experienced users, these logs are useful when troubleshooting problems with builds if required.
+
+. In the *Pipeline* page, click *View Log* for the build pipeline in progress.
+. When prompted, log into Jenkins with your OpenShift Online account. If the page does not display, you may need to wait for a few minutes for the Jenkins instance to initialize and try again.
```

Comment:
Probably my own fault, but there shouldn't be a "may" in our docs.

Reply:
Yep, may, could, were part of existing docs, but weeded them out now, thanks for pointing them out.